local/etc/algolia/docs_datadoghq_preview.json

```diff
@@ -136,9 +136,9 @@
     "selectors": {
         "api_latest": {
             "lvl0": {
-                "selector": "",
+                "selector": ".h-100 .sidenav-nav > ul >li.open > a",
                 "global": true,
-                "default_value": "API Latest"
+                "default_value": "API"
```

Comment:
I think you will need to add the same changes to `docs_datadoghq.json`

Reply:
thanks @davidejones!